src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/DirectiveAttributeCompletionItemProvider.cs

```diff
@@ -115,20 +119,30 @@ internal static ImmutableArray<RazorCompletionItem> GetAttributeCompletions(
 
         foreach (var (displayText, (attributeDescriptions, commitCharacters)) in attributeCompletions)
         {
-            var insertText = displayText;
+            var insertTextSpan = displayText.AsSpan();
 
             // Strip off the @ from the insertion text. This change is here to align the insertion text with the
             // completion hooks into VS and VSCode. Basically, completion triggers when `@` is typed so we don't
             // want to insert `@bind` because `@` already exists.
-            var startIndex = insertText.StartsWith('@') ? 1 : 0;
+            if (SpanExtensions.StartsWith(insertTextSpan, '@'))
```

Comment:
Why can't this just be used as an extension method?

Reply:
both of them are in System - not sure if there is a better way to address this, but this is super-weird.